docs/change-log.md

```diff
@@ -2,6 +2,25 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+
+## Product Update February 24th, 2023
+
+- **New:** Extended MariaDB and MySQL driver inputs with optional parameter `privilege list` that will be applied to the generated user.
```

Comment:
- what do you mean by "generated user"?
- Can we link to: https://docs.humanitec.com/integrations/resource-types/mysql, https://docs.humanitec.com/integrations/resource-types/mariadb

Reply:
That's what is written in the copy in UI, I don't know what it means:
<img width="487" alt="Screenshot 2023-02-24 at 09 39 11" src="https://user-images.githubusercontent.com/1139124/221132065-d467f496-6184-4dd7-a5c6-ea40b7fbed42.png">

we can

Reply:
Ah, I see. Makes sense.